test/WebSites/XmlFormattersWebSite/Controllers/SerializableErrorController.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Mvc;
+
+namespace XmlFormattersWebSite.Controllers
+{
+    public class SerializableErrorController : Controller
+    {
+        [HttpGet]
+        public IActionResult ModelStateErrors()
+        {
+            Exception exception = null;
+
+            try
+            {
+                throw new Exception("Error in executing the action");
```

Comment:
why can't this be ```var exception = new Exception()...``` instead of throwing and catching and then doing the same thing?